src/main/java/crazypants/enderio/machine/obelisk/attractor/GuiAttractor.java

```diff
@@ -67,6 +68,7 @@ protected void drawGuiContainerBackgroundLayer(float par1, int par2, int par3) {
         ColorUtil.getRGB(Color.white));
   }
 
+
```

Comment:
plz no extra empty lines. if you want to separate blocks of methods use a comment line